### ISSUE TITLE
Unsafe get, put and array access

### ISSUE THREAD

**Issue by sirinath** [bytedeco/javacpp#156]
Due to unsafe methods getting intensified it is faster than native. So perhaps it might be an idea to replace where appropriate with Unsafe use.

Also using Unsafe to access Java arrays is faster since there not no bound checking, memory fence and type checking.

Some native methods may be replaceable with pure Java with Unsafe.

**Comment by saudet** [comment]
Yes, the indexer package could be enhanced in many ways. As usual, contributions are welcome!
